docs/literate/src/files/getting_started_with_Trixi.jl

```diff
@@ -0,0 +1,287 @@
+#src # Getting started with Trixi.jl
+
+# This tutorial is intended for beginners in Julia and Trixi.jl.
+# After reading it, you will install Julia and Trixi on your PC and will be able to download files
+# from Trixi github, execute and edit them.
+# **Trixi.jl** is a numerical simulation framework for hyperbolic conservation laws 
+# written in [`Julia`](https://julialang.org/).
+# This means that Julia have to be installed on a PC to work with Trixi. 
+
+# ## Julia installation
+
+# Trixi works with the current stable Julia release. More information about Julia support can be
+# found in [`README`](https://github.com/trixi-framework/Trixi.jl#installation).
+# The most fully explained installation process can be found in
+# [`Julia installation instruction`](https://julialang.org/downloads/platform/).
+# But you can follow also our short installation instruction.
+
+# ### Windows
+
+# - Download Julia [`installer`](https://julialang.org/downloads/) for Windows. Make sure 
+#   that you choose the right version of installer (64-bit or 32-bit) according to your computer.
+# - Open the downloaded installer.
+# - Paste an installation directory path or find it using a file manager (select `Browse`).
+# - Select `Next`.
+# - Check the `Add Julia to PATH` to add Julia to the Environment Variables. 
+#   This makes possible to run Julia using Terminal from any directory only typing `julia`.
+# - Select `Next`, then Julia will be insalled.
+
+# Now you can verify, that Julia is installed:
+# - Type `Win+R` on a keyboard.
```

Comment:
```suggestion
# - Press `Win+R` on a keyboard.
```
To me, "type" is ambiguous (it could also mean that they should literally type "W i n + R")